setup.py

```diff
@@ -35,6 +35,11 @@ def local_file(name):
     ],
     install_requires=[
     ],
+    extras_require={
+        'dev': [
+            'flake8',
+        ],
+    },
```

Comment:
This particular change isn't strictly necessary for the Makefile stuff, but it's a bit more idiomatic. requirements.txt files are more often used for pinning versions in an application than for library dependencies.